products/sle/main.pm

```diff
@@ -754,6 +754,10 @@ elsif (get_var("VIRT_AUTOTEST")) {
     }
     if (get_var("VIRT_PRJ1_GUEST_INSTALL")) {
         loadtest "virt_autotest/guest_installation_run";
+        #Snapshots are supported on KVM VM Host Servers only
```

Comment:
Space after `#`, please.

Reply:
Get it, will update PR with this change as your mentioned.